src/main/kotlin/no/nav/klage/oppgave/api/controller/BehandlingController.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.klage.oppgave.api.controller
+
+import io.swagger.annotations.Api
+import io.swagger.annotations.ApiParam
+import no.nav.klage.oppgave.api.view.BehandlingEditedView
+import no.nav.klage.oppgave.config.SecurityConfiguration.Companion.ISSUER_AAD
+import no.nav.klage.oppgave.repositories.InnloggetSaksbehandlerRepository
+import no.nav.klage.oppgave.service.BehandlingService
+import no.nav.klage.oppgave.util.getLogger
+import no.nav.klage.oppgave.util.logBehandlingMethodDetails
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@Api(tags = ["kabal-api"])
+@ProtectedWithClaims(issuer = ISSUER_AAD)
+@RequestMapping("/klagebehandlinger")
```

Comment:
Stemmer denne nå? Skulle vært `ankebehandlinger`?

Reply:
Vi har vel bevisst valgt å ikke endre URLene ennå

Reply:
(Det å sette på vent er forøvrig nå ment å gjelde alle typer behandlinger, ikke bare anker)

Reply:
Ja, ble minnet på det :)